src/AutoUI/index.tsx

```diff
@@ -279,6 +307,33 @@ export const AutoUI = <T extends AutoUIBaseResource<T>>({
 			!!sdk?.tags,
 		[actions, sdk?.tags],
 	);
+
+	const internalOnChange = (
+		updatedFilters: JSONSchema[],
+		sortInfo: TableSortOptions<T> | null,
+		page: number,
+		itemsPerPage: number = 50,
```

Comment:
I think the typings should allow us to do:
```suggestion
		itemsPerPage: number,
```